docs/source/prototype.models.rst

```diff
@@ -91,3 +91,21 @@ hifigan_vocoder_v3
 ~~~~~~~~~~~~~~~~~~
 
 .. autofunction:: hifigan_vocoder_v3
+
+
+SQUIM_OBJECTIVE
```

Comment:
Could you use autosummary?

Reply:
it turns out the code change should also cover other models. Let me keep what it is and create another PR for using autosummary.